src/Microsoft.ML.Transforms/Text/doc.xml

```diff
@@ -0,0 +1,185 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<doc>
+  <members>
+
+    <member name="TextTransform">
+      <summary>
+        A transform that turns a collection of text documents into numerical feature vectors.
+        The feature vectors are normalized counts of (word and/or character) ngrams in a given tokenized text.
+      </summary>
+      <remarks>
+        The TextFeaturizer transform gives user one-stop solution for doing:
+        <list type="bullet">
+          <item>
+            <description>Language Detection</description>
+          </item>
+          <item>
+            <description>Tokenzation​</description>
+          </item>
+          <item>
+            <description>Text normalization</description>
+          </item>
+          <item>
+            <description>Predefined and custom stopwords removal.</description>
+          </item>
+          <item>
+            <description>Word-based or character-based Ngram and SkipGram extraction.​</description>
+          </item>
+          <item>
+            <description>TF, IDF or TF-IDF.</description>
+          </item>
+          <item>
+            <description>L-p vector normalization.​</description>
+          </item>
```

Comment:
Later in this document and I think in other documents you tended to do

```xml
<item><description>Foo</description></item>
```

vs.

```xml
<item>
    <description>Foo</description>
</item>
```

The first form might make this list a little less voluminous and easier to read.